doc/dev_guide/deply_noobaa_on_local_machine.md

```diff
@@ -0,0 +1,131 @@
+# Deploy NooBaa on Local Machine
+There are many ways to deploy NooBaa on k8s on you local machine, this guide refers to 2 options (the difference is in the configurations):
+1. Docker Desktop + Minikube
+2. Rancher Desktop
+
+The guide includes the steps:
+1. Configuration (Docker Desktop + Minikube or Rancher Desktop)
+2. Build images (noobaa-core and noobaa-operator) and optional step to push to a container registry (Quay or Docker Hub).
+3. Install NooBaa using NooBaa CLI.
+
+## 1) Configuration (First Time Only)
+
+### Docker Desktop + Minikube
+#### Change Your Docker Resource Preferences
+Click on docker logo -> preference… -> Resource (left choice in the menu): CPU: 6, Memory: 10 GB, SWAP: 1 GB, Disk: 80 GB
+
+#### Setting Your Minikube Configurations
+a. Start your cluster:
```

Comment:
I think it's worth mentioning the minikube dashboard
`minikube dashboard --url`